resources/web/wwi/ProtoManager.js

```diff
@@ -29,7 +29,7 @@ export default class ProtoManager {
       // generate list of exposed parameters (that will be tied to interface elements)
       for (const [parameterName, parameter] of this.proto.parameters) {
         parameter._view = this.#view;
-        this.exposedParameters.set(parameterName, parameter); // TODO: change key to parameter id ?
+        this.exposedParameters.set(parameterName, parameter);
```

Comment:
I might be wrong but I believe we don't use this anymore since we allow to edit sub-proto from the interface, so as long as `this.proto` is set to the root prior to populating the window (might be the case already) this shouldn't be needed